pyproject.toml

```diff
@@ -64,7 +64,8 @@ dependencies = [
 ] 
 
 [tool.hatch.envs.tool.scripts]
-license = 'python -m licenseheaders -t license_header.tmpl -o "SonarSource SA" -y 2011-2023 -n "Sonar Scanner Python" -E .py' 
+license = 'python -m licenseheaders -d src -t license_header.tmpl -o "SonarSource SA" -y 2011-2023 -n "Sonar Scanner Python" -E .py' 
+license_check = ["license", "git diff --name-only ./src", "git diff --quiet ./src"]
```

Comment:
Could a single `git diff --name-only --exit-code` do the same job as the combination between both commands?
Also, shouldn't we run the license checks and generation on both `src` and `tests` ?

Reply:
Yes good catch!